### PR TITLE
Review the make.md chapter

### PR DESCRIPTION
**Pull request by Marta-MM**
### List of changes proposed in this PR (pull-request)

* Changed ## An Introduction to Make to ## Why This is Useful
* Changed ## Prerequisites / recommended skill level to Prerequisites
* Checked for correct grammar

### What should a reviewer concentrate their feedback on?

- [ ] *Lorem ipsum dolor sit amet, consectetur adipiscing.*
- [x] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file.
- [x] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/master/README.md#contributors) in the README file: Marta-MM

### CODE REVIEW

File: book/website/reproducible-research/make.md [hunk 1]
```diff
@@ -27,7 +27,7 @@ the way to the pdf file of the paper!
 <small style="margin: 5pt auto; text-align: center; display: block;">A
 schematic for a research project that uses LaTeX.</small>
 
-## An Introduction to Make
+## Why This is Useful
```
Comment (malvikasharan) [inline book/website/reproducible-research/make.md:30]:
```suggestion
## An Introduction to Make
```

Comment (malvikasharan) [reply book/website/reproducible-research/make.md:30]: I will rever this one, as I think in the new template we are no longer using the "why this is useful" - apologies that haven't yet merged the new template PR. That should be my next PR review :D

**Review by malvikasharan — APPROVED** [review]
Thank you @Marta-MM! Please go ahead and merge it. 🎉

**Comment by malvikasharan** [comment]
@all-contributors please add @Marta-MM for bug fix

**Comment by Marta-MM** [comment]
Merge pull request #1538 
as per https://github.com/alan-turing-institute/the-turing-way/pull/1538#pullrequestreview-527070011

**Pull request merged**